titus-ext/aws/src/main/java/com/netflix/titus/ext/aws/AwsConfiguration.java

```diff
@@ -63,4 +63,17 @@
      */
     @DefaultValue("60000")
     long getIamRoleCacheTimeoutMs();
+
+    /**
+     * IAM role ARN to assume into to access AWS API for the data plane account. If not set, it is assumed that
+     * the control plane and the data plane run in the same account and no cross access is required.
+     */
+    @DefaultValue("")
+    String getDataPlaneRoleArn();
```

Comment:
Once we add IAM validation, the control plane will still need to know the data plane Role ARN even if it's in the same account because it would still need to assume into it to do validation.

Reply:
Good point. We will need more configuration parameters to control that. For now, I will keep this simple rule.